_posts/2017-10-23-call-for-candidates-joint-board.md

```diff
@@ -0,0 +1,54 @@
+---
+layout: post
+subheadline: "Call for Joint Carpentries Leadership"
+title: "Call for Nominations to Joint Board"
+teaser: "Stand for election to the joint board of the merged Carpentries organization"
+header:
+ image_fullwidth: "light-blue-wood-texture.jpg"
+categories:
+   - blog
+comments: true
+show_meta: true
+authors: ["Karen Cranston"]
+
+---
+
+As most of you know, Software Carpentry and Data Carpentry and merging into a
+new organization, provisionally called "The Carpentries". This new organization
+will officially begin on January 1, 2018, with a hugely talented staff and
+dedicated community already in place.
+
+This is an exciting time, and we are looking for people who want to  help direct
+the new organization by being an elected member of the Board of Directors of The
+Carpentries. The board will include both appointed and elected members in order
+to balance community engagement with needed expertise in leading a growing
+non-profit organization. For more information about the responsibilities and
+composition of the board, see [this
+issue](https://github.com/carpentries/2017Merger/blob/master/RFCs/RFC2.md), part
+of the merger RFC.  
+
+If you are a qualified instructor who has taught at least twice in the past two
+years, or have done non-teaching work for the community, you can both stand for
```

Comment:
Suggestion for clarificaiton and to reflect current SCF bylaws.

**Who can vote in the election and run for the Board?**

Following current [SCF bylaws](https://software-carpentry.org/blog/2014/12/scf-governance.html) current Carpentries members may vote and serve on the Board. Election or appointment to the board is currently limited to members.

The membership is made up of:
- Every qualified instructor who has taught at least two Software or Data Carpentry workshops in the past two calendar years.
- Anyone who has done 30 days or more work for the Carpentries in the past calendar year.
- Anyone who has, in the opinion of the Steering Committee, made a significant contribution in the past year.
- The signatory for a Silver, Gold or Platinum Member Organization